MediaBrowser.XbmcMetadata/Parsers/BaseNfoParser.cs

```diff
@@ -208,8 +208,8 @@ protected virtual void Fetch(MetadataResult<T> item, string metadataFile, XmlRea
 
         protected void ParseProviderLinks(T item, string xml)
         {
-            // Look for a match for the Regex pattern "tt" followed by 7 digits
-            var m = Regex.Match(xml, @"tt([0-9]{7})", RegexOptions.IgnoreCase);
+            // Look for a match for the Regex pattern "tt" followed by 7 or 8 digits
+            var m = Regex.Match(xml, @"tt(\d{7,8})", RegexOptions.IgnoreCase);
```

Comment:
I found this interesting point about using `\d` vs `[0-9]`, so I think we actually want `[0-9]` here: https://stackoverflow.com/a/16621778/1988326

Confirmed locally as well:
![image](https://user-images.githubusercontent.com/2626103/79690907-05290380-822b-11ea-86b1-57f04f65722c.png)

Reply:
We should probably change all regexes that use `\d` and document his somewhere

Reply:
Is this within the scope of the PR though? I agree though @randrey we should probably leave it as `[0-9]` for this reason, unless this change in behaviour is part of the fix.

Reply:
Changing all other regexes should ofc be done in a separate pr

Reply:
Created #2923 to track updating all other regexes.

Reply:
That's an interesting observation. 
I've updated the code in those two files to use [0-9].